doc/dune-files.rst

```diff
@@ -1324,6 +1327,21 @@ For instance:
                  (((action (run ./pp.sh X=1 %{input-file})) foo bar))
                  (((action (run ./pp.sh X=2 %{input-file})) baz))))
 
+.. _compat-pp:
+
+Compat preprocessor
+~~~~~~~~~~~~~~~~~~~
+
+The ``compat`` preprocessing specification is equivalent to
+``no_preprocessing`` when using a recent versions of the
```

Comment:
```suggestion
``no_preprocessing`` when using a recent version of the
```

(should this be "the latest"?)

Reply:
note that it can be several versions of the compiler

Reply:
I change this sentence